source/stylesheets/base/_navigation.scss

```diff
@@ -12,35 +12,52 @@
   .container {
     display: flex;
     flex-direction: row;
+    position: relative;
 
-    img {
-      transition: height 0.1s ease;
+    .blog-title {
+      color: $red;
+      font-size: 87px;
+      line-height: 80px;
+      margin-bottom: 0.3em;
+      transform: rotate(-8deg);
+      transition: font-size 0.1s ease;
+    }
+
+    .blog-title-first-line {
+      padding-left: 15px;
     }
 
     h3 {
       flex: 1 0 0;
     }
 
     ul {
-      align-content: space-evenly;
-      align-items: flex-end;
-      background: url("../images/nav_background.svg") no-repeat top right;
+      bottom: 20px;
       display: flex;
-      flex: 2 0 0;
-      justify-content: flex-end;
       list-style: none;
+      position: absolute;
```

Comment:
There's a position mixin in Bourbon you can make use of:

```
@include position(absolute, null 0 20px null);
```

https://github.com/thoughtbot/bourbon/blob/master/core/bourbon/library/_position.scss